standardization/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -18,18 +18,22 @@ package za.co.absa.enceladus.standardization.interpreter.stages
 import org.apache.spark.sql.types._
 import org.apache.spark.sql._
 import org.apache.spark.sql.functions._
-import za.co.absa.enceladus.utils.types.{Defaults, Format}
+import za.co.absa.enceladus.utils.types.Defaults
+
 import scala.util.Random
 import za.co.absa.enceladus.utils.error.ErrorMessage
 import za.co.absa.enceladus.utils.schema.SchemaUtils.appendPath
 import za.co.absa.enceladus.utils.error.UDFLibrary
 import za.co.absa.spark.hofs._
+
```

Comment:
What is the reason for adding newlines here in such a way?

Reply:
None. I blame the IDE. I rarely meddle with `import` directly except when removing unused items.

Reply:
My wrong, function declaration is not counted into the limit. Please consider my previous comment moot. 😊